### PR TITLE
CI config changes to speed up tests

### PR DESCRIPTION
**Pull request by chrisbennight**
Updated accumulo 1.5.x version to 1.5.2

### CODE REVIEW

**Comment by chrisbennight** [comment]
Hmm, not sure why this can't be merged automatically - I rebased against master.  Might me something with the formatting cleanup of the Pom.   Since that is hard to read, changes were
  *increment default versions of
    *geotools
    *geoserver
    *accumulo
    *hadoop
    *jacoco
  *move hortonworks and cloudera profiles over to their respective versions/builds of accumulo
  *travis.yml changes
    *make document generation only run on master
    *forked coveralls plugin tests first to see if it is in the local maven cache before clone and install
    *document generation phase moved to before install so it gets included in the travis repo cache

File: pom.xml [hunk 1]
```diff
@@ -210,197 +210,199 @@
...
+                <configuration>
+                    <jacocoReports>
+                        <jacocoReport>${project.basedir}/geowave-index/target/site/jacoco-ut/jacoco.xml</jacocoReport>
+                        <jacocoReport>${project.basedir}/geowave-analytics/target/site/jacoco-ut/jacoco.xml
+                        </jacocoReport>
+                        <jacocoReport>${project.basedir}/geowave-ingest/target/site/jacoco-ut/jacoco.xml</jacocoReport>
+                        <jacocoReport>${project.basedir}/geowave-raster/target/site/jacoco-ut/jacoco.xml</jacocoReport>
+                        <jacocoReport>${project.basedir}/geowave-store/target/site/jacoco-ut/jacoco.xml</jacocoReport>
+                        <jacocoReport>${project.basedir}/geowave-types/target/site/jacoco-ut/jacoco.xml</jacocoReport>
+                        -
```
Comment (rwgdrummer) [inline pom.xml]: Do we need this '-'? Can I remove it.  If so, I have the rebased  version ready to push and then merge.

**Comment by rwgdrummer** [comment]
See my comment in the diff.  I see the changes.  I am ready to purge, but I need to verify the one comment.

**Comment by rwgdrummer** [comment]
See my comment in the diff.
Please verify so I can complete the merge.

**Comment by chrisbennight** [comment]
Nope, don't need the dash- not sure where it creeped in

**Pull request merged**